### PR TITLE
Lk contribution guidelines

### PR DESCRIPTION
**Pull request by ekiernan**
Added new contribution docs and guidelines.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Contributing to WDL-docs
+
+WDL strives to be community driven and thus relies on suggestions and assistance from people such as yourself. We welcome any contribution that you are willing to provide whether it be in the form of syntax suggestions, feature requests, documentation help, or simply pointing out our ~~tyops~~ typos. All that we ask is that you agree to the following [contributor agreement](#contributor-agreement) and follow a [few guidelines](#how-to-contribute).
```
Comment (kayleemathews) [inline CONTRIBUTING.md:3]:
```suggestion
WDL strives to be community driven and thus relies on suggestions and assistance from people such as yourself. We welcome any contribution that you are willing to provide whether it be in the form of syntax suggestions, feature requests, documentation help, or simply pointing out our ~~tyops~~ (typos). All that we ask is that you agree to the following [contributor agreement](#contributor-agreement) and follow a [few guidelines](#how-to-contribute).
```

File: README.md [hunk 2]
```diff
@@ -2,9 +2,50 @@
...
+* Typo fixes
+* Updates to doc content
+* Suggestions for existing docs
+* New docs that you want to add to the repo
+* New WDL resources that you want to list or link to
+
+## How to contribute
+There are two main ways you can contribute:
+1. [File an issue](https://github.com/openwdl/wdl-docs/issues/new/choose) in wdl-docs repository (good option if you're lacking time to make a fix). 
+2. Make a fork of the repository to add updates or new docs and make a PR.
```
Comment (kayleemathews) [inline README.md:15]:
```suggestion
2. Make a fork of the repository to add new docs or update existing ones and make a PR.
```

File: README.md [hunk 3]
```diff
@@ -2,9 +2,50 @@
...
+There are multiple branches in the wdl-docs repo. Since documentation will differ between WDL spec versions, each set of documentation representing a particular spec should live on a branch labeled by the spec version with major.minor.path versioning. For example, documentation for WDL 1.0 lives on the `1.0.0` branch. This allows you to specify which version of the documentation you want to the read on the wdl-docs site. 
+
+The wdl-docs **main** branch is used for the latest version of WDL for which we have documentation, which is currently WDL 1.0. This branch is deployed to the readthedocs website's `latest` version for documentation. 
+
+The wdl-docs site also has a stable version of documentation which currently points to the `1.0.0` branch. This will likely change in the future.  
+
+### Site map in the mkdocs yml
+The docs folder contains a [mkdocs yml](https://github.com/openwdl/wdl-docs/blob/main/mkdocs.yml) file which contains the site map for the documentation website. All the website sections and article names are organized using this yml file. If you want to add new sections to the website or add or change article names, use this yml file.
```
Comment (kayleemathews) [inline README.md:33]:
```suggestion
### Site map in the mkdocs YML
The docs folder contains a [mkdocs YML](https://github.com/openwdl/wdl-docs/blob/main/mkdocs.yml) file which contains the site map for the documentation website. All the website sections and article names are organized using this YML file. If you want to add new sections to the website or add or change article names, use this YML file.
```

File: README.md [hunk 4]
```diff
@@ -2,9 +2,50 @@
...
+
+## Contribution process
```
Comment (kayleemathews) [inline README.md:35]: I think this list of steps is helpful - thanks for including it!

File: docs/index.md [hunk 5]
```diff
@@ -1,18 +1,56 @@
...
+* Typo fixes
+* Updates to doc content
+* Suggestions for existing docs
+* New docs that you want to add to the repo
+* New WDL resources that you want to list or link to
+
+## How to contribute
+There are two main ways you can contribute:
+1. [File an issue](https://github.com/openwdl/wdl-docs/issues/new/choose) in wdl-docs repository (good option if you're lacking time to make a fix). 
+2. Make a fork of the repository to add updates or new docs and make a PR.
```
Comment (kayleemathews) [inline docs/index.md:22]:
```suggestion
2. Make a fork of the repository to add new docs or update existing ones and make a PR.
```

File: docs/index.md [hunk 6]
```diff
@@ -1,18 +1,56 @@
...
+
+### Branches for different WDL versions
+There are multiple branches in the wdl-docs repo. Since documentation will differ between WDL spec versions, each set of documentation representing a particular spec should live on a branch labeled by the spec version with major.minor.path versioning. For example, documentation for WDL 1.0 lives on the `1.0.0` branch. This allows you to specify which version of the documentation you want to read on the wdl-docs site. 
+
+The wdl-docs **main** branch is used for the latest version of WDL for which we have documentation, which is currently WDL 1.0. This branch is deployed to the readthedocs website's `latest` version for documentation. 
+
+The wdl-docs site also has a stable version of documentation which currently points to the `1.0.0` branch.  
+
+### Site map in the mkdocs yml
+The docs folder contains a [mkdocs yml](https://github.com/openwdl/wdl-docs/blob/main/mkdocs.yml) file which contains the site map for the documentation website. All the website sections and article names are organized using this yml file. If you want to add new sections to the website or add or change article names, use this yml file.
```
Comment (kayleemathews) [inline docs/index.md:41]:
```suggestion
### Site map in the mkdocs YML
The docs folder contains a [mkdocs YML](https://github.com/openwdl/wdl-docs/blob/main/mkdocs.yml) file which contains the site map for the documentation website. All the website sections and article names are organized using this YML file. If you want to add new sections to the website or add or change article names, use this YML file.
```

**Review by kayleemathews — APPROVED** [review]
This looks great Liz. Thanks for working on it!

File: README.md (hunk 4 above)
Comment (ekiernan) [reply README.md:35]: Thanks for taking a look!

**Review by vdauwera — APPROVED** [review]
This is all great! My only comment is that there is some duplicated content between the readme and the docs index, which may lead to maintenance burden in future so you might want to resolve that somehow, but it's certainly not a blocking issue and shouldn't stand in the way of getting this out.

**Comment by ekiernan** [comment]
Thanks

> 

Thank you for taking a look! We will definitely want to make the update process simpler. I updated the index.md so that it matches the readme with the exception of a readthedocs specific comment about finding the link to the document an the top two sections.

**Pull request merged**